extensions/amp-analytics/amp-analytics.md

```diff
@@ -164,6 +164,12 @@ Type attribute value: `quantcast`
 
 Adds support for Quantcast Measurement. More details for adding Quantcast Measurement can be found at [quantcast.com](https://www.quantcast.com/help/guides/)
 
+##### SimpleReach
+
+Type attribute value: `simplereach`
+
+Adds support for SimpleReach.  Configuration details can be found at [simplereach.com/docs](http://docs.simplereach.com/dev-guide/implementation/google-amp-implementation)
```

Comment:
We can deploy that page as early as tomorrow.  You can view the staging version here: http://docs.staging-001.simplereach.com.s3-website-us-east-1.amazonaws.com/dev-guide/implementation/google-amp